apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -289,7 +273,9 @@ NetSimLocalClientNode.prototype.disconnectRemote = function (onComplete) {
 
     self.myWire = null;
     // Trigger an immediate router update so its connection count is correct.
-    self.myRouter.update(onComplete);
+    self.myRouter.update(function (err, result) {
+      onComplete(result);
+    });
```

Comment:
This and the change above it seem to be a couple more places where our callback doesnt take an error. Might be good to have it use callback(err, result) just for consistency, even if we'll never send it an error.